py_ts_interfaces/parser.py

```diff
@@ -39,6 +40,14 @@ class PossibleInterfaceReference(str):
 PreparedInterfaces = Dict[str, InterfaceAttributes]
 
 
+def camelize(s):
+    """Convert a string from underscore naming convention to camel-case.
+    """
+    # Force a character to be at the start, so that strings starting with underscore
+    # do not get that initial underscore removed.
+    return re.sub(r'(.)_([a-z])', lambda x: x.group(1) + x.group(2).upper(), s)
```

Comment:
This appears to fail if a string has 2+ underscores.

Current behavior:
```
a_b_c  ->  aB_c
a_b_c_d -> aB_cD
```

In general this is a great feature, and I'd like to merge it, but I think we could use some unit tests.  We _could_ test this `camelize` method, but preferably we would test in a parametrized way the interface, using the `test_parser_flush` found in `tests.py` as a template for a new test for flushing with camelize turned on.

Reply:
I added some tests and fixed the behavior you saw (it only failed with one letter words between the underscores so I didn't notice it). I added a test for the conversion with multiple underscores, because I am not sure what the output should be. Personally I prefer to have "word__word" convert to "word_Word" but I'm not sure it's the expected behavior for everyone.